features/git-town-new-pull-request/self_hosted.feature

```diff
@@ -0,0 +1,45 @@
+Feature: git-town new-pull-request: when origin is a self hosted servie
+
+  As a developer trying to create a pull request when using a self hosted service
```

Comment:
I like the new job story format:

```
When creating a pull request on a self-hosted repository whose type cannot be derived from its URL
I want to specify the type of the hosted repository
So that Git Town can support specific features of my repository platform.